src/View/Helper/AbstractHelper.php

```diff
@@ -558,4 +559,25 @@ protected function hasAllowedPrefix(string $attribute): bool
 
         return false;
     }
+
+    /**
+     * translate the label
+     */
+    protected function translateLabel(string|int $label): string|int
```

Comment:
Let's make this new methods `@internal`, I don't want to add new APIs as `protected` methods in extensible classes are

Reply:
done